### PR TITLE
log content of outgoing emails

### PR DESCRIPTION
**Pull request by yaitskov**
Feature SES

### CODE REVIEW

File: localstack/services/ses/ses_starter.py [hunk 1]
```diff
@@ -25,6 +27,8 @@ def email_responses_send_raw_email(self):
 
         raw_data = to_str(base64.b64decode(self.querystring.get('RawMessage.Data')[0]))
 
+        LOGGER.error('Raw email:\n%s' % raw_data)
```
Comment (whummer) [inline localstack/services/ses/ses_starter.py:30]: Error log level seem a bit too verbose here - can we use `LOGGER.info` instead?

**Review by whummer — COMMENTED** [review]
Thanks for this PR @yaitskov - only added a minor comment. Also, can we please check the linter issue in this failing build here: https://travis-ci.org/github/localstack/localstack/builds/750887742 Thanks!

File: localstack/services/ses/ses_starter.py (hunk 1 above)
Comment (yaitskov) [reply localstack/services/ses/ses_starter.py:30]: Doh... I wanted debug.

Comment (yaitskov) [reply localstack/services/ses/ses_starter.py]: fixed

**Comment by whummer** [comment]
LGTM, thanks @yaitskov ! 👍

**Pull request merged**